test/lib/mayaUsd/render/mayaToHydra/testMtohDagChanges.py

```diff
@@ -271,7 +272,7 @@ def test_node_creation_undo(self):
 
         cmds.undoInfo(state=0)
         try:
-            cmds.file(new=1, f=1)
+            mayaUtils.openNewScene()
```

Comment:
This test hasn't been enabled in over a year; see [the test CMakeLists](https://github.com/Autodesk/maya-hydra/blob/63334cf1d91c140f0fc1b598872d05a2b3a276b9/test/lib/mayaUsd/render/mayaToHydra/CMakeLists.txt#L5-L7). I updated it on principle in case we bring it back and based on how I think it should be updated, but I haven't actually run it.